README.md

```diff
@@ -261,7 +282,7 @@ Tests
 =====
 
 
-**Run unit-tests:**
+**Run unit-tests**
```

Comment:
Not sure this is relevant in the top level README, maybe we should just link to tests/README.md, or let people find it themselves.

Reply:
completely agree, was thinking the same. same in .net. this was just supposed to be a quick iterative change, small PR, merge quickly, that sort of thing. but since you bring it up, i'll reorg it.

Reply:
the README.md should be completely focused on users, so i moved all this stuff to a new DEVELOPER.md doc.